site/docs/foundations/characteristics/index.mdx

```diff
@@ -6,42 +6,46 @@ sidebar:
   priority: 1
 ---
 
-## About Characteristics
+A design token stores design decisions in a single place, enabling teams to use them to create consistent experiences.
 
-Characteristic design tokens are a level of abstraction sitting between the foundational tokens and the component-specific tokens.
+The Salt Design System uses three tiers of tokens. From most generalized to most specific, they are:
 
-Each grouping of characteristics (also known in other design systems as _semantics_) is a set of CSS variables which encapsulates a specific concept. These tokens can then be reused by multiple components where necessary, allowing for consistency across design and logical, reusable patterns.
+1. Foundational tokens
+2. Characteristic design tokens (characteristics)
+3. Component-specific tokens
 
-### What are characteristics?
+Characteristics form sets of Cascading Style Sheet (CSS) variables that define a visual attribute, such as text or colors denoting status. Components can then reuse them, allowing for design consistency and logical patterns. They help to define the rules that style each component and aid the cohesiveness of the overall user interface.
 
-Characteristics are semantic groupings of visual attributes that are applied across relevant components. They drive consistency and support theming across complex libraries. They help to define the rules that style each component and aid the cohesiveness of the UI.
+## Benefits
 
-### Why characteristics?
+For designers, characteristics enable visual changes without impacting the efficiency or effectiveness of components.
 
-From a design perspective, there is a need for aesthetic flexibility without decreasing the efficiency of build or the effectiveness of the components when implemented. From a dev perspective, it provides an easy way to reuse the same values, without the overhead of having to remember details such as hex values. Any time you want to tweak the value of a characteristic, you only have to do it in one place for it to reflect everywhere.
+For developers, they offer an easy way to reuse the same values. It creates a single point of reference for commonly used values, which means you only have to update in one place for it to reflect everywhere.
```

Comment:
Hmm... the first line about designers feels a bit woolly. What's it actually saying? It's not clear how visual changes without tokens impact the "efficiency or effectiveness of components".

As for the second point - that benefit also applies to designers.

The following bullet list summarises the benefits quite well IMHO, so perhaps these 2 paragraphs aren't needed at all?

That being said, one additional benefit I'd call out (perhaps as an additional bullet below?) is that (characteristic) design tokens create a shared language between designers, devs and other stakeholders - they all use the same names to refer to the same aspects/concepts in Salt-based UIs. That helps avoid miscommunications and streamlines design-to-dev handovers too.